bin/run-canonical-extensions-tests.sh

```
@@ -0,0 +1,106 @@
+	file='/tmp/WOOCOMMERCE_CANONICAL_EXTENSIONS'
+	echo -n 'Fetching extensions list: ';
+	# The variable can be actualized under https://github.com/woocommerce/woocommerce/settings/variables/actions (mix of public and private repository URLs)
+	( gh variable get CANONICAL_EXTENSIONS > $file && echo 'done' ) || ( echo 'error' && exit 1 )
+	repositories=( $( cat $file | tr -d '\r' | tr '\n' ' ' ) )
+fi
+
+# Sort out which repositories provide the necessary workflows first.
+filtered=()
+skipped=()
+echo -n "Looking up repositories (${#repositories[@]}): "
+for repository in ${repositories[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)